cvmfs/swissknife_zpipe.cc

```diff
@@ -26,7 +26,7 @@
 #include "duplex_zlib.h"
 #include "swissknife_zpipe.h"
 
-#if defined(MSDOS) || defined(OS2) || defined(WIN32) || defined(__CYGWIN__)
```

Comment:
Needs to be reverted

Reply:
sorry about that, good job spotting! I will just exclude `msdos` entirely.